build/pipelines/fuzz.yml

```diff
@@ -0,0 +1,58 @@
+trigger: none
+#  batch: true
+#  branches:
+#    include:
+#      - main
+#  paths:
+#    exclude:
+#      - docs/*
+#      - samples/*
+#      - tools/*

```

Comment:
### Discussion
I left these on here. Not sure if we felt ready to run this regularly off of `main` just yet. Let me know what you think.